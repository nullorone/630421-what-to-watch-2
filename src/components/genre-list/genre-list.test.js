import React from "react";
import renderer from "react-test-renderer";
import GenreList from "./genre-list";

describe(`Make snapshot`, () => {
  const initProps = {
    genres: [`Academy`, `Javascript`],
  };

  it(`Get snapshot component`, () => {
    const tree = renderer
      .create(
          <GenreList
            {...initProps}
          />
      )
      .toJSON();

    expect(tree).toMatchSnapshot();
  });
});
