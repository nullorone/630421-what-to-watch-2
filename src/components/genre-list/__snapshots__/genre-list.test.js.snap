// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Make snapshot Get snapshot component 1`] = `
<ul
  className="catalog__genres-list"
>
  <li
    className="catalog__genres-item"
  >
    <a
      className="catalog__genres-link"
      href="#"
    >
      Academy
    </a>
  </li>
  <li
    className="catalog__genres-item"
  >
    <a
      className="catalog__genres-link"
      href="#"
    >
      Javascript
    </a>
  </li>
</ul>
`;
